prepare/cards/amazon_mass.py

```diff
@@ -0,0 +1,53 @@
+from datasets import get_dataset_config_names
+
+langs = get_dataset_config_names("AmazonScience/massive")
+# now langs is the list of all languages showing in the dataset
+
+from datasets import load_dataset_builder
+
+ds_builder = load_dataset_builder("AmazonScience/massive")
+classlabels = ds_builder.info.features["intent"]
```

Comment:
Hi. This is something that lends itself to a development worth adding to unitxt.

Create an Operator that can be used to extract all values of a specific field.

ExtractMostCommonFieldValues(from_stream="train", from_field="intent",to_field = "choices")

Reply:
discussed and put into issue #297